learn/best_practices/arrays.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: book
+title: Arrays
+permalink: /learn/best_practices/arrays
+---
+
+When passing arrays in and out of a subroutine/function, use the
+following pattern for 1D arrays (it is called <span
+class="title-ref">assumed-shape</span>):
```

Comment:
I think this can be simply formatted as:
```suggestion
following pattern for 1D arrays (it is called `assumed-shape`):
```

Reply:
OK, I have just pushed a commit with Markdown italics (as in Fortran90.org) for those five HTML tags:
```bash
$ grep -inRI "</span>"
arrays.md:9:class="title-ref">assumed-shape</span>):
arrays.md:47:<span class="title-ref">explicit-shape</span> arrays:
arrays.md:55:To use <span class="title-ref">explicit-shape</span> arrays, do:
element_operations.md:10:-   <span class="title-ref">explicit-shape</span> arrays
element_operations.md:102:Or one can use <span class="title-ref">explicit-shape</span> arrays as
```